modules/ossm-security-cert-manage.adoc

```diff
@@ -12,6 +12,7 @@ By default, {ProductName} generates a self-signed root certificate and key and u
 * Install {ProductName} with mutual TLS enabled to configure certificates.
 * This example uses the certificates from the link:https://github.com/maistra/istio/tree/maistra-{MaistraVersion}/samples/certs[Maistra repository]. For production, use your own certificates from your certificate authority.
 * Deploy the Bookinfo sample application to verify the results with these instructions.
+* Install OpenSSL to verify certificates.
```

Comment:
I would change this to "OpenSSL is required to verify certificates" (because it might already be installed, so no need to install it again).

Reply:
made edit